### DISCUSSION TITLE
Can't toggle breakpoint

### DISCUSSION THREAD

**Discussion by flippette** [helix-editor/helix#10283] [Troubleshooting]
### Summary

I can't toggle breakpoints in any file with space-d-b. Launch (space-d-l) works fine though.

### Reproduction Steps

 

I tried this:

1. `hx file.rs`
2. \<type out some code\>
3. `space-d-b`

I expected this to happen:
The line has a breakpoint annotation beside it.

Instead, this happened:
Nothing happens.

### Helix log

I deleted the log file hoping for it to fill up, nothing shows up.

### Platform

Arch Linux

### Terminal Emulator

kitty 0.26.4

### Helix Version

helix 22.08.1

**Comment by CptPotato** [comment]
- On a lot of themes inactive breakpoints are barely visible, can you double check if that's the case?
  ![image](https://user-images.githubusercontent.com/3957610/197710140-fe70bbb4-fa07-4054-bed3-8baaeed8e8cf.png)
- Breakpoints use this symbol `⊚`, maybe the terminal doesn't display it correctly?
- Do other ways to set the breakpoint work?
  - Clicking the gutter/line number
  - Calling the `Toggle breakpoint` command from the command palette (<kbd>Space</kbd> <kbd>?</kbd>)

**Comment by flippette** [comment]
- Clicking the gutter _does_ work, but the command palette command doesn't.
- I'm using gruvbox on kitty.
- It seems that the breakpoint glyph is displayed on 2 cells, and the left one is for some reason highlighted?
![image](https://user-images.githubusercontent.com/43138244/197753758-70fffea2-f3aa-464d-9929-b7285cf8512f.png)

**Comment by CptPotato** [comment]
It's strange that neither <kbd>Space</kbd> <kbd>d</kbd> <kbd>b</kbd> nor the command palette work, but clicking the gutter does.
Maybe one of the authors that worked on the DAP integration can chime in?

The visual difference isn't a big issue I think, depending on the terminal emulator (and maybe font?) some glyphs are drawn two cells wide.
